### DISCUSSION TITLE
Personalities for Escorts

### DISCUSSION THREAD

**Discussion by yafflehk** [endless-sky/endless-sky#9174] [Ideas]
Given that personalities already exist for ships in the game, can they be assigned to escorts? It would be a simple way to implement a kind or formation flying and would give the player more options for controlling large fleets. It might also go some way towards avoiding the "Battle ball" that large engagements often turn into. Some types that occurred to me were:
Wingman: Sticks with the player ship no matter what, similar to the Group command now
Wingman, ranging: The current default escort ship behavior
Aggressive: Roams the system looking for fights
Escorts: Sticks with and protects ships the player is escorting
Hang Back: Waits beyond the system boarder
Allies: Protects ships that are allies of the player
Landers: Goes straight to the target/nearest landable plant and orbits it
Screen: Gets between the player and any enemy ships
Miner: Mines asteroids

**Comment by lumbar527** [comment]
A few things:

1. I would like this.
2. It's probably been discussed before, not sure though. Maybe in more of a mining situation, with all the discussions of fleet mining.
3. I think "Aggressive" should be "Patrolling" instead.

**Comment by kybermen** [comment]
To choose which "personality", maybe add different Outfits that could change the personality, Like a "Tactic Cards" or something.
The problem would be allowing only one outfit to get one personality.

**Comment by SpaceCaptainBill** [comment]
Maybe add "Tactic Chips" and "Tactic Chip Ports"

Maybe only advanced/human ships can have tactics cards.
